src/reflect/scala/reflect/internal/Symbols.scala

```diff
@@ -3017,7 +3017,7 @@ trait Symbols extends api.Symbols { self: SymbolTable =>
       res
     }
 
-    override def isVarargs: Boolean = definitions.isVarArgsList(paramss.flatten)
+    override def isVarargs: Boolean = definitions.isVarArgsList(lastNonEmpty(paramss))
```

Comment:
Eh. I think I've seen this before. I'm not sure how I feel about the condition here: why is
```scala
def bleh(i: Int*)(): Int
```
considered varargs, but
```scala
def bleh(i: Int*)(nyeh: String): Int
```
not?

Imagine `nyeh` as an implicit parameter and it is even less justifiable, IMO.

What uses this flag?

Reply:
It appears only to be exposed in the `api.MethodSymbolApi`, where it is documented as "Does this method support variable length argument lists?"

It's not really useful to know the answer to that question without knowing which param list is varargs.

`api.TermSymbolApi` really ought to expose `isVarargs` that you would call on the parameter symbol itself.